README-MR.md

```diff
@@ -22,54 +22,54 @@
 </details>
 <!-- Do not translate this -->
 
-# Welcome newbie Open Source contributors!
+# आपले स्वागत आहे नवख्या ओपन सोर्स सहयोगी!
 
 [![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
 [![first-timers-only Friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://www.firsttimersonly.com/)
 [![Build Status](https://api.travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source.svg?branch=master)](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source)
 
-This is a list of resources for people who are new to contributing to Open Source.
+ओपन सोर्समध्ये सहयोग देण्यासाठी नवीन असलेल्या लोकांच्या संसाधनांची ही सूची आहे.
 
-If you find additional resources, please contribute a pull request.
+आपल्याला अतिरिक्त संसाधने आढळल्यास, कृपया पुल विनंतीस सहयोग द्या.
 
-If you have questions or comments, please create an issue.
+आपल्याकडे प्रश्न किंवा टिप्पण्या असल्यास कृपया एक समस्या तयार करा.
 
-**Contents**
+**सामग्री**
 
-- [Contributing to Open Source in general](#contributing-to-open-source-in-general)
-- [Direct GitHub searches](#direct-github-searches)
-- [Mozilla's contributor ecosystem](#mozillas-contributor-ecosystem)
-- [Useful articles for new Open Source contributors](#useful-articles-for-new-open-source-contributors)
-- [Using Version Control](#using-version-control)
-- [Open Source books](#open-source-books)
-- [Open Source contribution initiatives](#open-source-contribution-initiatives)
-- [License](#license)
+- [सर्वसाधारणपणे ओपन सोर्समध्ये योगदान देणे](#सर्वसाधारणपणे-ओपन-सोर्स-स्त्रोत-योगदान-देणे)
+- [डायरेक्ट गिटहब शोध](#डायरेक्ट-गीथब-सर्च)
+- [मोझिलाची योगदानकर्ता पर्यावरणीय प्रणाली](#मोज़िला-योगदानकर्ता-इकोसिस्टम)
+- [नवीन मुक्त स्त्रोत योगदानकर्त्यांसाठी उपयुक्त लेख](#नवीन-मुक्त-स्त्रोत-योगदानकर्त्यांसाठी-उपयुक्त-लेख)
```

Comment:
I think this would sound better - नवीन open-source योगदानकर्त्यांसाठी उपयुक्त लेख